sdks/git-sdk/src/test/java/org/jreleaser/sdk/git/ChangelogGeneratorUnitTest.java

```diff
@@ -0,0 +1,116 @@
+package org.jreleaser.sdk.git;
+
+
+import javafx.util.Pair;
```

Comment:
This causes build failures as JavaFX is not in the classpath.
You'll have to supply your own `Pair` class or use one provided by Mockito or Junit (if any)